storage/types.go

```diff
@@ -469,6 +469,29 @@ func (n *NodeID) Suffix(prefixBytes, suffixBits int) *Suffix {
 	return NewSuffix(byte(b), sfxPath)
 }
 
+// Prefix returns a NodeID's prefix.
+// This is the same value that would be returned from Split, but without the
+// overhead of calulating the suffix too.
```

Comment:
Prob worth documenting that it returns a copy. And possibly in Split().

Reply:
👍